docs/testplan/snappi-tests/capacity_test.md

```diff
@@ -0,0 +1,91 @@
+# SONiC Switch Capacity Test
+
+
+- [SONiC Switch Capacity Test](#sonic-switch-capacity-test)
+  - [Test Objective](#test-objective)
+  - [Test Setup](#test-setup)
+  - [Test Steps](#test-steps)
+  - [Label Structure](#label-structure)
+    - [Common labels for all metrics](#common-labels-for-all-metrics)
+    - [Metric labels](#metric-labels)
+      - [PSU Metrics](#psu-metrics)
+      - [Sensor Temperature Metrics](#sensor-temperature-metrics)
+      - [Queue Metrics](#queue-metrics)
+      - [Interface Metrics](#interface-metrics)
+
+## Test Objective
+
+This test aims to assess the true capacities of SONiC switches.
+
+## Test Setup
+
+The test is designed to be topology-agnostic, meaning it does not assume or impose a specific setup wiring, settings, or configuration. The running script will track power usage, temperature, queue watermark, and health state of all SONiC switches in a testbed.
+
+## Test Steps
+
+1. Run all traffic generators at full line rate, injecting traffic into SONiC devices to maximize stress. The test duration is configurable, with a default setting of 15 minutes.
+2. Using the switch commands or RPC (Remote Procedure Call), collect all SONiC switches’ metrics listed in the next section. The data sampling rate is also configurable with a default value of every 1 minute.
+3. Save the sampled raw data to a database via the telemetry interface provided by the SONiC team in test_reporting folder. The metrics are structured as data points in our database. An example of how to use the interface is provided in telemetry folder.
```

Comment:
We need to be specific on which interface to use. In this case, we are expecting periodic one being used.